include/experimental/__p0009_bits/config.hpp

```diff
@@ -198,23 +198,12 @@ static_assert(_MDSPAN_CPLUSPLUS >= MDSPAN_CXX_STD_14, "mdspan requires C++14 or
 #endif
 
 #ifndef _MDSPAN_USE_CLASS_TEMPLATE_ARGUMENT_DEDUCTION
-// GCC 10's CTAD seems sufficiently broken to prevent its use.
-#  if (defined(_MDSPAN_COMPILER_CLANG) || !defined(__GNUC__) || __GNUC__ >= 11) \
-      && ((defined(__cpp_deduction_guides) && __cpp_deduction_guides >= 201703) \
-         || (!defined(__cpp_deduction_guides) && MDSPAN_HAS_CXX_17))
+#  if (defined(__cpp_deduction_guides) && __cpp_deduction_guides >= 201703) \
```

Comment:
@adah1972 Thank you for the contribution!  Have you tested this with GCC 8?  I've found that GCC 8 sometimes doesn't build C++17 code that works fine with GCC 7.5 or GCC 9.  The automated PR tests don't include a GCC 8 test.

Reply:
> @adah1972 Thank you for the contribution! Have you tested this with GCC 8? I've found that GCC 8 sometimes doesn't build C++17 code that works fine with GCC 7.5 or GCC 9. The automated PR tests don't include a GCC 8 test.

I am happy to report that all tests compile and pass with the current patch under GCC 8.5.0.

Reply:
Thanks for testing this!